presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetCompressionUtils.java

```diff
@@ -45,23 +47,22 @@ public static Slice decompress(CompressionCodecName codec, Slice input, int unco
             case GZIP:
                 return decompressGzip(input, uncompressedSize);
             case SNAPPY:
-                return decompressSnappy(input, uncompressedSize);
+                return decompress(new SnappyDecompressor(), input, uncompressedSize);
+            case LZO:
+                return decompress(new LzoDecompressor(), input, uncompressedSize);
```

Comment:
did u check the LZO decompressor is working? Was doing this way, but seeing some wired errors
We could enable LZO in testcase to guard this